### PR TITLE
PAGE_ShowHeader accepts raw strings

### PR DESCRIPTION
**Pull request by howard0su**

### CODE REVIEW

File: src/pages/128x64x1/model_page.c [hunk 1]
```diff
@@ -152,7 +152,7 @@ void PAGE_ModelInit(int page)
     memset(gui, 0, sizeof(struct modelpage_obj));
     mp->file_state = 0;
     mp->last_txpower = Model.tx_power;
-    PAGE_ShowHeader(_tr("Model setup")); // using the same name as related menu item to reduce language strings
+    PAGE_ShowHeader(_tr_noop("Model setup")); // using the same name as related menu item to reduce language strings
```
Comment (PhracturedBlue) [inline src/pages/128x64x1/model_page.c]: Lint Error:
```
155:  At least two spaces is best between code and comments  
```

File: src/pages/128x64x1/standard/curves_page.c [hunk 2]
```diff
@@ -106,7 +106,7 @@ static void show_page(CurvesMode _curve_mode, int page)
     }
 
     set_cur_mixer();
-    //PAGE_ShowHeader(_tr("Mode"));
+    //PAGE_ShowHeader(_tr_noop("Mode"));
```
Comment (PhracturedBlue) [inline src/pages/128x64x1/standard/curves_page.c]: Lint Error:
```
109:  Should have a space between // and comment  
```

File: src/pages/128x64x1/telemconfig_page.c [hunk 3]
```diff
@@ -80,7 +80,7 @@ void PAGE_TelemconfigInit(int page)
         return;
     }
 
-    PAGE_ShowHeader(_tr("Telemetry config")); // using the same name as related menu item to reduce language strings
+    PAGE_ShowHeader(_tr_noop("Telemetry config")); // using the same name as related menu item to reduce language strings
```
Comment (PhracturedBlue) [inline src/pages/128x64x1/telemconfig_page.c]: Lint Error:
```
83:  At least two spaces is best between code and comments  
```

File: src/pages/128x64x1/trim_page.c [hunk 4]
```diff
@@ -66,7 +66,7 @@ static int row_cb(int absrow, int relrow, int y, void *data)
 
 static void _show_page()
 {
-    //PAGE_ShowHeader(_tr("Trim")); // no title for devo10
+    //PAGE_ShowHeader(_tr_noop("Trim")); // no title for devo10
```
Comment (PhracturedBlue) [inline src/pages/128x64x1/trim_page.c]: Lint Error:
```
69:  Should have a space between // and comment  
```

File: src/pages/128x64x1/model_config.c [hunk 5]
```diff
@@ -93,9 +93,9 @@ void PAGE_ModelConfigInit(int page)
 {
     (void)page;
     switch (Model.type) {
-      case MODELTYPE_HELI: show_titlerow(_tr("Helicopter")); break; 
-      case MODELTYPE_PLANE: show_titlerow(_tr("Airplane")); break; 
-      case MODELTYPE_MULTI: show_titlerow(_tr("Multirotor")); break; 
+      case MODELTYPE_HELI: show_titlerow(_tr_noop("Helicopter")); break; 
+      case MODELTYPE_PLANE: show_titlerow(_tr_noop("Airplane")); break; 
+      case MODELTYPE_MULTI: show_titlerow(_tr_noop("Multirotor")); break;
```
Comment (PhracturedBlue) [inline src/pages/128x64x1/model_config.c]: Lint Error:
```
96:  Line ends in whitespace.  Consider deleting these extra spaces.  
97:  Line ends in whitespace.  Consider deleting these extra spaces.  
98:  Line ends in whitespace.  Consider deleting these extra spaces.  
```

File: src/pages/common/standard/_common_standard.c [hunk 6]
```diff
@@ -94,7 +94,7 @@ const char *STDMIX_TitleString(guiObject_t *obj, const void *data)
     (void)obj;
     int pageid = ((unsigned long)data) & 0xFFFF;
     int sw     = (((unsigned long)data) >> 16) & 0xFFFF;
-    sprintf(tempstring, "%s - ", PAGE_GetName(pageid));
+    sprintf(tempstring, "%s - ", _tr(PAGE_GetName(pageid)));
```
Comment (PhracturedBlue) [inline src/pages/common/standard/_common_standard.c]: Lint Error:
```
97:  Never use sprintf. Use snprintf instead.  
```

File: src/pages/320x240x16/trim_page.c [hunk 7]
```diff
@@ -82,7 +82,7 @@ void PAGE_TrimEditInit(int page)
     tp->index = page;
     tp->trim = trim[tp->index];
 
-    //PAGE_ShowHeader(_tr("Trim"));
+    //PAGE_ShowHeader(_tr_noop("Trim"));
```
Comment (PhracturedBlue) [inline src/pages/320x240x16/trim_page.c]: Lint Error:
```
85:  Should have a space between // and comment  
```

**Comment by PhracturedBlue** [comment]
FYI, you can run 'make lint' on your own machine.  It should tell you about linting issues compared to the 'master' branch.  So as long as you keep master synced with upstream, it should match what the Travis build does.  If not, let me know and I'll try to fix it.

**Comment by howard0su** [comment]
I missed the part of sync master with latest. It works fine after that.

**Pull request merged**